### DISCUSSION TITLE
Cannot login in initial state of Galaxy Z flip 5

### DISCUSSION THREAD

**Discussion by SyuA0529** [community/community#68967] [Mobile]
### Select Topic Area

Bug

### Body

In initial state of Galaxy Z Flip 5, I cannot login  github by application
I login github by browser and redirection to application, but nothing work
When I update Android OS, now I can login successful

**Comment by elisealix22** [comment]
Thank you so much for reporting this! We recently published [an updated FAQ](https://gh.io/mobilefaq) around sign in that we hope will help people troubleshoot issues. I'm glad you were able sign in and please let us know if you continue to have any issues.

**Comment by nour-eldein10** [comment]
1Try to use different engines like chrome safari firefox brave opera maybe the problem of some web browser
2if it's not supported for the device try to download it from other way like Chrome or any browser and download it that support the ios and manipulation to the device system by trying other ways far from system
3 if all that doesn't work operate a stimulation or virtual device as another android device like virtual box in windows for Linux you can make too android virtual device
